apps/calendar/js/views/hour_double_tap.js

```diff
@@ -40,12 +40,40 @@ HourDoubleTap.prototype = {
     this.alldaysHolder.removeEventListener('click', this._onAllDayTap);
   },
 
-  _onDayTap: function(evt) {
+  _isEventRightOnButton(evt) {
     var target = evt.target;
     if (!target.classList.contains('md__day')) {
+      // Could be aggressive event fuzzing registering a tap on an existing
+      // event or the "add event" UI, where the tap is really below that item.
+      // See bug 1210201.
+
+      // In this case, offsetY is the vertical space between the top of the
+      // element and the actual tap, so if the tap is bigger than the height
+      // of the element, treat it as an md__day tap.
+      var tappedOnExistingEvent =
+        target.classList.contains('md__event-like-button') &&
+        evt.offsetY >= 0 &&
+        evt.offsetY <= target.getBoundingClientRect().height;
+
+      if (tappedOnExistingEvent) {
+        return true;
+      }
+    }
+
+    return false;
+  },
+
+  _onDayTap: function(evt) {
+    if (this._isEventRightOnButton(evt)) {
       return;
     }
 
+    var target = evt.target.closest('.md__day');
+    // Note that until Bug 1091889 gets fixed we'll still get the :active
+    // styles because they're being applied with `touchstart` and we can't
+    // defaultPrevent this event without breaking everything.
+    evt.preventDefault();
```

Comment:
I can see placing the preventDefault as an else for the `if (this._isEventRightOnButton(evt))` block, since it is tied specifically to why it is checked to be a fuzzed event. If the event fuzzing concern goes away, and that if test is no longer needed, it would be good if the preventDefault had a chance to be removed at that time too, indicated by grouping/proximity.